### PR TITLE
add mirrord operator status docs

### LINKED ISSUE #242: Add docs on using mirrord operator status
Should probably be under `using mirrord`

### PR DESCRIPTION
**Pull request by aviramha**
Closes #242

### CODE REVIEW

File: content/docs/using-mirrord/status.md [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+```
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+| Session ID       | Target                      | Namespace | User                                                          | Ports | Session Duration |
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+| 487F4F2B6D2376AD | deployment/ip-visit-counter | default   | Aviram Hassan/aviram@metalbear.co@avirams-macbook-pro-2.local |       | 4s               |
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+```
+
+The User field is generated in the following format - whoami/k8s-user/hostname. whoami is from the local machine, and the hotname, while k8s user is the user we see from the operator side.
```
Comment (Razz4780) [inline content/docs/using-mirrord/status.md:22]:
```suggestion
The User field is generated in the following format - whoami/k8s-user/hostname. whoami and hostname are from the local machine, while k8s user is the user we see from the operator side.
```

**Review by Razz4780 — CHANGES_REQUESTED** [review]

**Review by Razz4780 — APPROVED** [review]

**Review by Anita-ihuman — CHANGES_REQUESTED** [review]
Left a few comments here worth considering

File: content/docs/using-mirrord/status.md [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+date: 2024-02-20T11:59:39+01:00
+lastmod: 2024-02-20T11:59:39+01:00
+draft: false
+weight: 150
+toc: true
+tags: ["team", "enterprise"]
+---
+
+When the operator is installed as part of mirrord for Teams, users can use the command `mirrord operator status` to see active sessions in the cluster.
+For example, in the following output, we can see the session ID, the target used, the namespace of the target, the session duration, and the user running that session. We can also see that `Ports` is empty meaning, the user isn't stealing or mirroring any traffic at the moment.
```
Comment (eyalb181) [inline content/docs/using-mirrord/status.md:13]:
```suggestion
For example, in the following output, we can see the session ID, the target used, the namespace of the target, the session duration, and the user running that session. We can also see that `Ports` is empty, meaning the user isn't stealing or mirroring any traffic at the moment.
```

File: content/docs/using-mirrord/status.md [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+```
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+| Session ID       | Target                      | Namespace | User                                                          | Ports | Session Duration |
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+| 487F4F2B6D2376AD | deployment/ip-visit-counter | default   | Aviram Hassan/aviram@metalbear.co@avirams-macbook-pro-2.local |       | 4s               |
++------------------+-----------------------------+-----------+---------------------------------------------------------------+-------+------------------+
+```
+
+The `User` field is generated in the following format - `whoami/k8s-user/hostname`. `whoami` and `hostname` are from the local machine, while `k8s-user` is the user we see from the operator side.
```
Comment (eyalb181) [inline content/docs/using-mirrord/status.md:22]:
```suggestion
The `User` field is generated in the following format - `whoami/k8s-user@hostname`. `whoami` and `hostname` are from the local machine, while `k8s-user` is the user we see from the operator side.
```

**Review by eyalb181 — CHANGES_REQUESTED** [review]

**Review by Anita-ihuman — APPROVED** [review]

**Review by eyalb181 — APPROVED** [review]

**Pull request merged**